### DISCUSSION TITLE
[Projects Beta] Time value in Project Insights isn't useful

### DISCUSSION THREAD

**Discussion by jpike88** [community/community#34913] [Projects and Issues]
I want to make a chart using a field 'effort points' with type Number as my Y axis, and 'Time Closed' as my X axis value.

So I made the new field today, and change the effort points for a bunch of issues going back in the last 6 months, almost all of them were closed a while ago.

Instead, the current "Time" value looks to care about WHEN I applied the effort points, not the date closed of the issues. This renders the Time column kind of useless right now.

See screenshots.
![Screen Shot 2022-10-03 at 10 19 11](https://user-images.githubusercontent.com/9585787/193495721-025f880f-c5c2-4044-bd2a-9c8d23a4851d.png)
![Screen Shot 2022-10-03 at 10 19 20](https://user-images.githubusercontent.com/9585787/193495733-c686eead-b685-403c-8fcb-a2c69b6e9ee5.png)

**Comment by jd-santos** [comment]
I just spent a ton of time categorizing my issues into specific labels to create a graph for my executive team, but it was all a waste because of this issue. They all only show up as today's date.
